nginx/conf.d/geoip2.conf

```diff
@@ -11,3 +11,9 @@ geoip2 /usr/share/GeoIP/GeoLite2-ASN.mmdb {
     $ip_aso      source=$remote_addr autonomous_system_organization;
     $ip_as_build_epoch metadata build_epoch;
 }
+
+geoip2 /usr/share/GeoIP/GeoLite2-City.mmdb {
+    auto_reload 1d; # Automatically reload the database daily
```

Comment:
We don't need those additional comments just for the city database, so let's maintain consistency here.